src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -442,4 +446,50 @@ private boolean checkAcceptedBody(String accept) {
         }
         return false;
     }
+
+    /**
+     * Build the request URI for the future consumer requests
+     *
+     * @param routingContext context of the current HTTP request
+     * @return the request URI for the future consumer requests
+     */
+    private String buildRequestUri(RoutingContext routingContext) {
+        String baseUri = routingContext.request().absoluteURI();
+
+        String xForwarded = routingContext.request().getHeader("x-forwarded");
```

Comment:
It's just forwarded, right?

Reply:
damn, confused by the others starting with `X-`